WcaOnRails/app/models/delegate_report.rb

```diff
@@ -18,6 +18,7 @@ def equipment_default
     self.equipment = "Gen 2 Timer: 0
 Gen 3 Pro Timer: 0
 Gen 4 Pro Timer: 0
+Did you attach O-rings (or something similar) to the timers to prevent pressing buttons by accident? If not, please explain why.
```

Comment:
That suggests that people might attach O-rings to only one button. Why would or could someone do that?

Reply:
I could see someone just covering the reset button since all the talk is about "timer resets". I guess we wouldn't know for sure unless we explicitly ask about covering both of them.

Alternatively, we could ask a more generic question like: "Did you make any modifications to the timer? If you did, please explain what modifications you made, and if you did not, please explain why you did not".

Reply:
The power button also causes a reset. ;-)

I think we should aim to have a default answer (and ask if people deviated from that default), so that it's easy for Delegates to answer, and for a reader to see if it's the default answer. 

I think O-rings (or similar) over both buttons is a good default, since that is a safe future state to aim for.

Reply:
Our gen 3 pro timers are equipped with only o ring on the reset button, basically because the power button is hard enough to reach.

Reply:
Hmm... I can't remember that. :confused:
Anyway, although it's unlikely to reach the power button with your thumb when stopping, it can be pushed by a puzzle falling on the button. I think that's how at least some resets are caused.

Reply:
This conversation has convinced me that we should explicitly mention both the power button *and* the reset button. I've updated my PR with a new wording.